docs/kcl/std.json

```diff
@@ -153550,25 +153550,79 @@
     "args": [
       {
         "name": "num",
-        "type": "i64",
+        "type": "NumberArg",
         "schema": {
           "$schema": "https://spec.openapis.org/oas/3.0/schema/2019-04-02#/definitions/Schema",
-          "title": "int64",
-          "type": "integer",
-          "format": "int64"
+          "title": "NumberArg",
+          "description": "A number that can be either a floating point number or an integer.",
```

Comment:
Can't we just say "A number between -3e32 and 9e28 inclusive" ? (Not real values, only an example)

If the range is too difficult to express, it may be better to simply say "A rational number".

A floating point is a programmer term.